services/provider/server/server.go

```diff
@@ -509,8 +509,12 @@ func (s *OCSProviderServer) appendClientProfileExternalResources(
 		profileMap[rbdClientProfileName] = rbdClientProfile
 
 	}
+	rnsName := consumerConfig.GetRbdRadosNamespaceName()
+	if rnsName == util.ImplicitRbdRadosNamespaceName {
+		rnsName = util.ImplicitRbdRadosNamespaceNameCephFormat
+	}
 	rbdClientProfile.Rbd = &csiopv1a1.RbdConfigSpec{
-		RadosNamespace: consumerConfig.GetRbdRadosNamespaceName(),
+		RadosNamespace: rnsName,
```

Comment:
I am not sure what is this about

Reply:
informed offline, for implicit rns the spec here should be an empty string or we can choose to not set anything